workers/social/lib/social/models/oauth.coffee

```diff
@@ -129,23 +131,83 @@ module.exports = class OAuth extends bongo.Base
     # -- GITHUB PROVIDER --------------------------------------------------8<--
 
     github    :
-      title   : 'Github OAuth Provider'
-      enabled : false
+      title   : 'GitHub OAuth Provider'
+      enabled : yes
+      scopes  : ['user', 'user:email', 'user:follow', 'public_repo', 'repo',
```

Comment:
some of the scopes here are redundant and i believe some other are not required, why did you add all the scopes?